src/backend/egl/context.rs

```diff
@@ -56,8 +56,8 @@ impl EGLContext {
         assert!(!config_id.is_null(), "EGL configuration id pointer is null");
         assert!(!context.is_null(), "EGLContext pointer is null");
 
-        let display = EGLDisplay::from_raw(display, config_id)?;
-        let pixel_format = display.get_pixel_format(config_id)?;
+        let display = unsafe { EGLDisplay::from_raw(display, config_id) }?;
+        let pixel_format = unsafe { display.get_pixel_format(config_id) }?;
```

Comment:
Is `get_pixel_format` declared unsafe? If it is we should take a look if we can lift that requirement, that function just returns an enum after all.